proteomic_aggregation/python/proteomic_aggregation.ipynb

```diff
@@ -0,0 +1,3591 @@
+{
```

Comment:
<p>Might want to add a sentence that tells the reader where the term razor comes from for this.  It's such an odd word, but once one says - "like Occam's razor", it really helps summarize the essence of the term.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>